Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.yml

```diff
@@ -656,6 +675,13 @@ script:
       - text
       required: false
       secret: false
+    - auto: PREDEFINED
+      defaultValue: 'true'
+      description: Whether to download the report immediately or not. If the report is taking more than 10 seconds to be ready, set as *false*.
```

Comment:
If true, downloads the report immediately after the report is generated. The default is "true". If the report takes longer than 10 seconds to generate, set to "false".